backend/src/main/java/com/example/backend/repo/ToDoRepo.java

```diff
@@ -0,0 +1,46 @@
+package com.example.backend.repo;
+
+
+import com.example.backend.model.ToDo;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Repository
+public class ToDoRepo {
+
+    private Map<String, ToDo> toDos = new HashMap<>(
+            Map.of(
+                    "1", new ToDo("Wash the car", "OPEN", "1")));
+
+    public List<ToDo> getAllToDos(){
+        return new ArrayList<ToDo>(toDos.values());
+    }
+
+    public ToDo getById(String id) {
+                return toDos.get(id);
+            }
+
+    public ToDo addNewToDo(ToDo toDo) {
+        toDos.put(toDo.getId(), toDo);
+        return toDo;
+    }
+
+    public ToDo changeToDo(ToDo toChangeToDo) {
+        toDos.get(toChangeToDo.getId()).setStatus(toChangeToDo.getStatus());
+        toDos.get(toChangeToDo.getId()).setDescription(toChangeToDo.getDescription());
```

Comment:
Die put() Methode der Map überschreibt das bisherige Element.

Vorschlag: 
```suggestion
        toDos.put(toChangeToDo.getId(), toChangeToDo);
```